src/app/main.py

```diff
@@ -21,7 +21,17 @@ async def root():
     return "Hello World"
 
 
-@app.post("/expenses/", response_model=schemas.Expense)
+@app.get("/expenses/", response_model=list[schemas.Expense])
+async def read_expenses():
+    return expenses
+
+
+@app.get("/expenses/{expense_id}/", response_model=schemas.Expense)
+async def read_expense(expense_id):
+    return expenses[expense_id]
```

Comment:
You can leverage FastAPI's `HTTPException` class for this check. You will want to make sure to add the import at the top of the file.
```python
from fastapi import FastAPI, HTTPException
```
And then the actual check will be this:
```suggestion
async def read_expense(expense_id):
    if expense_id not in expenses:
        raise HTTPException(status_code=404, detail="Expense not found")
    return expenses[expense_id]
```